### ISSUE TITLE
[TF 2.0 API Docs] tf.math.add

### ISSUE THREAD

**Issue by josephhaaga** [tensorflow/tensorflow#25846]
<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>

**System information**
- TensorFlow version: 2.0
- Doc Link: https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/math/add

**Describe the documentation issue**

Much like #25802, documentation for `tf.math.add` is created from a generated file `python/ops/gen_math_ops.py`; a link to the file that generates `python/ops/gen_math_ops.py` would be handy for users.

`tf.math.add` could use a usage example, and a list of Errors raised. 

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**
Yes

**Comment by Sangarshanan** [comment]
Hey beginner here, On which branch can I find tf v2.0 ? cause it seems to end with 1.99

**Comment by jvishnuvardhan** [comment]
@Sangarshanan you can download from [here](https://pypi.org/project/tf-nightly-2.0-preview/)

**Comment by MarkDaoust** [comment]
>@Sangarshanan you can download from [here](https://pypi.org/project/tf-nightly-2.0-preview/)

Yes that's the pip package.

The package is built from `master` with a 2.0 config flag... I think @dynamicwebpaige knows the exact commands. Paige is someone working on a install/source page for the tf2 docs?

> a link to the file that generates `python/ops/gen_math_ops.py` would be handy for users.

Yes! 
I believe it's [this file](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/math_ops.cc#L409), but I can't think of a way to find this by introspection while generating the docs. Anyone have ideas?

( I usually find them by searching for quotes from the tf.org page )

> `tf.math.add` could use a usage example, and a list of Errors raised.

@josephhaaga can you send a PR?

**Comment by dynamicwebpaige** [comment]
@MarkDaoust @Sangarshanan @josephhaaga 
You can build TF 2.0 from source with:

```
export TF_NEED_CUDA=0
cd tensorflow
yes "" | ./configure

bazel build --config=opt --config=v2 tensorflow/tools/pip_package:build_pip_package
mkdir pip_pkg
./bazel-bin/tensorflow/tools/pip_package/build_pip_package pip_pkg --nightly_flag \
  --project_name tf_nightly_2.0_preview

```
I do not think that anyone is working on adding installation from source to the docs - but that would be a great add! So would the guide to modifying `gen_math_ops.py` documentation. :smile:

Thanks for your interest in improving the TF 2.0 API docs! We appreciate it.

**Comment by igormp** [comment]
I believe this could be closed due to #33773

**Comment by MarkDaoust** [comment]
yes.
